src/examples/usb-midi/main.go

```diff
@@ -16,12 +15,11 @@ func main() {
 	led.Configure(machine.PinConfig{Mode: machine.PinOutput})
 
 	button := machine.BUTTON
-	button.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
+	button.Configure(machine.PinConfig{Mode: machine.PinInputPulldown})
```

Comment:
I think there is an optimal pullup/pulldown choice for each target, but it would be confusing to use pulldown only for this example.
I hope we can find some way to make it better.

Reply:
```
$ git grep -i pininput ./src/examples/
src/examples/button/button.go:  button.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/button2/button2.go:        button1.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/button2/button2.go:        button2.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/button2/button2.go:        button3.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/button2/button2.go:        button4.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/hid-keyboard/main.go:      button.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/hid-mouse/main.go: button.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
src/examples/pdm/pdm.go:        machine.BUTTONA.Configure(machine.PinConfig{Mode: machine.PinInputPulldown})
src/examples/pininterrupt/circuitplay-express.go:       buttonMode      = machine.PinInputPulldown
src/examples/pininterrupt/pca10040.go:  buttonMode      = machine.PinInputPullup
src/examples/pininterrupt/stm32.go:     buttonMode      = machine.PinInputPulldown
src/examples/pininterrupt/wioterminal.go:       buttonMode      = machine.PinInput
src/examples/usb-midi/main.go:  button.Configure(machine.PinConfig{Mode: machine.PinInputPullup})
```

Reply:
It may be possible to create a default setting for `machine.BUTTON`, I do not know.

Reply:
The button question is a bit beyond the scope of this PR, I think. No?

Reply:
Yes, it is.
So I think it is better not to change it in this PR.
However, since there is no direct problem, either is fine.